SU-SWS/saa_victoria_subtheme
# Adapt 1679  event page h1

# READY FOR REVIEW

# Summary
- Apply correct font size to h1 on events pages

# Review By (Date)
- 2/1/2021

# Urgency
- How critical is this PR? Medium/high but small, but required for update today before Racial Justice launch next monday

# Steps to Test

1. Look at https://saauserguide-test.sites.stanford.edu/events and https://saauserguide-test.sites.stanford.edu/news and compare with a random regular page: https://saauserguide-test.sites.stanford.edu/card-tests
2. Inspect h1 sizes, make sure they are all the same. They should match the regular page at 2.44140625em


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-1679 part 2
- Other PRs ADAPT-1679 already merged
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
